### DISCUSSION TITLE
json to mask png

### DISCUSSION THREAD

**Discussion by GRF-Sunomikp31** [wkentaro/labelme#1395] [Q&A / Help]
Hi，I would like to ask if there is any code for converting json file into png mask？ While a pixel can have multiple semantic labels, the code provided by labelme can only have one semantic label per pixel

**Comment by wkentaro** [comment]
@GRF-Sunomikp31 Is this what you want?
https://gist.github.com/wkentaro/c76b8e40ea4b3150509bfc97b6281879
